oam/install_scripts/mcs-controllernode.service.in

```diff
@@ -5,6 +5,7 @@ After=mcs-workernode.service
 
 [Service]
 Type=forking
+ExecStartPre=/usr/bin/env bash -c "systemctl start mcs-workernode"
```

Comment:
Fixes mcs-columnstore restart not starting mcs-workernode again. Let's ensure mcs-workernode is active